src/reachy2_sdk/parts/head.py

```diff
@@ -152,8 +156,9 @@ def rotate_to(
         """
         if duration == 0:
             raise ValueError("duration cannot be set to 0.")
-        if self.is_off():
-            raise RuntimeError("Head is off. Rotate_to not sent.")
+        if not self.neck.is_on():
```

Comment:
why not if self.neck.is_off()? it's easier to understand

Reply:
Because is_off() does not exist for orbitas, as it is either on or off (contrary to parts that can be partially on and off). Do you want me to create it?

Reply:
Oups sorry I thought it was the is_on of Head. All good then!